e2e/rbd.go

```diff
@@ -487,6 +516,13 @@ var _ = Describe("RBD", func() {
 						go func(w *sync.WaitGroup, n int, p v1.PersistentVolumeClaim, a v1.Pod) {
 							name := fmt.Sprintf("%s%d", f.UniqueName, n)
 							wgErrs[n] = createPVCAndApp(name, f, &p, &a, deployTimeout)
+							filePath := appClone.Spec.Containers[0].VolumeMounts[0].MountPath + "/test"
+							checkSumClone := ""
+							checkSumClone, wgErrs[n] = calculateSHA512sum(f, &a, filePath, &opt)
```

Comment:
Because you run this in a loop of go-routines, each call to `calculateSHA512sum()` would need to know the name of the app to use. Maybe you need to pass `name` to the function too?

Reply:
yes, will have a look into this and update the PR